Documentation/WritingReST/CheatSheet.rst

```diff
@@ -337,6 +337,21 @@ Source::
       $a = 'hello';
 
 
+Placeholders
+------------
+
+Use the *Backus-Naur form* `<placeholder-name>` for placeholders in code and
+URLs.
+
+Examples:
+
+#. :php:`class <Entity>Controller extends ActionController`
```

Comment:
IMO we should remove this since we'd usually use a full and obvious name like `ExampleController` instead. Also In PHP code this looks like [generics](https://wiki.php.net/rfc/generics) and thus even more confusing.

Reply:
Ok, if this looks to much like generics, but instead of removing could you replace it with another PHP code example?

Reply:
Sure, e.g. said `ExampleController`.

Reply:
Yes, but there is no placeholder in that example then.

Reply:
Correct, because it does not make sense to have one here. Code does rarely use placeholders because it already has something similar: variables. Or static identifiers like class names. These by themselves already serve as placeholders.

Reply:
So you say, we should not add a PHP example as there are no ways to use placeholders in PHP in a useful way? I think, the whole Convention-over-Configuration setting in Extbase is a very good example of using placeholders. To explain the naming schema of a model, its controller and its repository, you can either use an real world example or just describe it in this templating way. And to not confuse the reader you can pick up the placeholder in your text flow and tell what it is about.

Reply:
As discussed we can keep this as example. We'll see if it really leads to confusion.